kernel/aster-nix/src/resource/mod.rs

```diff
@@ -0,0 +1,4 @@
+// SPDX-License-Identifier: MPL-2.0
+
+//! Resource control and accounting module.
+pub mod c_type;
```

Comment:
Does `rusage` will be used by other syscalls?

If so, add `resource` as a module under `aster-nix/util`, they are commonly-used structures among maybe many syscalls.

If not, just define `rusage` in `wait4.rs`.